argostime/graphs.py

```diff
@@ -76,12 +76,19 @@ def generate_price_graph_data(offer: ProductOffer) -> str:
 
         sales_dates.append((start, end))
 
+    title_size = 24
```

Comment:
Voor de leesbaarheid is het denk ik wel fijn om met wat comments te verhelderen waarom je dit precies doet